specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/servers.json

```diff
@@ -436,14 +442,6 @@
         },
         "kind": {
           "description": "Kind of sql server. This is metadata used for the Azure portal experience.",
-          "enum": [
```

Comment:
Removed because it was incorrect, kind might be a semicolon separated list like "v12.0;user" so it shouldn't be an enum.